study.md

```diff
@@ -112,13 +119,14 @@ From a collection of galaxies, update the `name` attribute of the record with
 the `designation` attribute of "NGC 224" to "Andromeda".
 
 ```ruby
-# your answer here
+Galaxy.where(designation:'NGC 224').update_all(name:"Andromeda")
```

Comment:
Be careful when writing something like this because this will update every record that gets returned with `#where` rather just than just one particular record.